### DISCUSSION TITLE
「HELP」failed to negotiate protocol: protocols not supported: [/test/1.0.0]

### DISCUSSION THREAD

**Discussion by WHOLETTHEDOG-OUT** [libp2p/go-libp2p#3065] [Q&A] [answered]
I use host.Connect(context.Background(), *info) will succuess,
but 
while doing host.NewStream() , there's always failed:
"failed to negotiate protocol: protocols not supported: [/test/1.0.0]"

**Comment by sukunrt** [comment, marked as answer]
The peer doesn't speak `/test/1.0.0`.
